test/battle/ability/parental_bond.c

```diff
@@ -286,6 +286,26 @@ SINGLE_BATTLE_TEST("Parental Bond Snore strikes twice while asleep")
     }
 }
 
+SINGLE_BATTLE_TEST("Parental Bond will block hit switch target effects before the second hit")
```

Comment:
```suggestion
SINGLE_BATTLE_TEST("Parental Bond only triggers Dragon Tail's target switch out on the second hit")
```

Reply:
You can just commit and merge test name changes

Reply:
Is that ok for your PRs? I didn't do it because I wanted the suggestions to go through you first before merging.

Reply:
If it is just comment changes I don't mind. I'm not good at making descriptions so I'm sure the suggestion is better then anything I came up with

Reply:
Ye, speaking for myself here, not others :sweat_smile: